src/main/java/me/xhyrom/spawnergenz/SpawnerGenz.java

```diff
@@ -79,6 +76,7 @@ else if (action.containsKey("sound"))
         getServer().getPluginManager().registerEvents(new ClickListener(), this);
         getServer().getPluginManager().registerEvents(new SpawnerListener(), this);
         getServer().getPluginManager().registerEvents(new BlockListener(), this);
+        new Hooks();
```

Comment:
just preference, dont we want to have `Hooks#init` static than creating class? That's just preference but there's no reason  for not using static functions

Reply:
tell your opinion